x/pairing/client/cli/tx_unstake_provider.go

```diff
@@ -54,6 +58,11 @@ func CmdUnstakeProvider() *cobra.Command {
 					return err
 				}
 				msgs = append(msgs, msg)
+
+				revokeGrantFeeMsg := CreateRevokeFeeGrantMsg(clientCtx, chainID)
```

Comment:
do we need to check if there is a feegrant at all?

Reply:
done. see https://github.com/lavanet/lava/commit/af338548b5293fffb5736dfc1b4c246893ce6c0b